### ISSUE TITLE
Aff4 support

### ISSUE THREAD

**Issue by spacepatcher** [log2timeline/plaso#2759]
Hi there!

Is it planned to add global processing of input images in [aff4](https://www.aff4.org/) format?

**Comment by joachimmetz** [comment]
Yes, https://github.com/log2timeline/dfvfs/issues/362. We are blocked on pyaff4 https://github.com/aff4/pyaff4/issues/9 at this point
